secure_message/resources/drafts.py

```diff
@@ -90,7 +88,7 @@ class DraftList(Resource):
     def get():
         """Get message list with options"""
 
-        string_query_args, page, limit, ru_id, survey, cc, label, desc, ce = get_options(request.args)
+        string_query_args, page, limit, _, _, _, _, _, _ = get_options(request.args)
```

Comment:
It would be nicer if get_options returned a dict or was removed as this looks like the only place it's used

Reply:
Agreed. It is used elsewhere but also probably beyond the scope of this PR to refactor the utility.